main/integration-tests/__test__/api-tests/indexes/create-index.test.js

```diff
@@ -49,10 +49,9 @@ describe('Create index scenarios', () => {
       });
     });
 
-    it('should fail if projectId is duplicate to the one already in the system', async () => {
+    it('should fail if indexId is duplicate to the one already in the system', async () => {
       const admin2Session = await setup.createAdminSession();
-      const testProj = await adminSession.resources.projects.mustFind(setup.gen.defaultProjectId());
-      const defaultIndexId = testProj.indexId;
+      const defaultIndexId = setup.defaultIndexId;
```

Comment:
Please see my comment earlier regarding the use of setup.defaults.indexId , etc.  It would be great if we start this pattern.

Reply:
As discussed, I'll cover the `defaults` pattern in my next resource test suite development.